app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestViewModel.kt

```diff
@@ -45,15 +52,21 @@ class MeetingRequestViewModel(
     viewModelScope.launch {}
```

Comment:
This `init` block can be removed since it doesn't launch anything

Reply:
Yes you are right, i ll remove it